README.md

```diff
@@ -65,7 +65,7 @@ These can be combined like so:
 
 ## Learning More About Jekyll and Liquid Used
 
-We have a styleguide which you can access at [http://localhost:7331/web/styleguide/](http://localhost:7331/web/styleguide/) that should get you up and running with a lot of the custom and useful info.
+We have a styleguide which you can access at [http://localhost:7331/web/styleguide/](http://localhost:7331/web/resources/styleguide/) that should get you up and running with a lot of the custom and useful info.
```

Comment:
Need to update the link text in between the [] as it still points to the old address.